bin/node/runtime/src/lib.rs

```diff
@@ -1626,4 +1626,14 @@ mod tests {
 
 		is_submit_signed_transaction::<Runtime>();
 	}
+
+	#[test]
+	fn call_size() {
+		assert!(
+			core::mem::size_of::<Call>() <= 200,
+			"size of Call is more than 200: some calls have too big arguments, use Box to reduce the
+			size of Call.
+			If the limit is too strong, maybe consider increase the limit to 300.",
+		);
+	}
```

Comment:
have you considered making a `static_assertion!` instead?

Reply:
I gave a try but I think the error message is better with a regular assert.
But all in all this assert is not really important there is no important issue with using a big call enum

```
error[E0080]: evaluation of constant value failed
    --> bin/node/runtime/src/lib.rs:1632:3
     |
1632 | /         static_assertions::const_assert!(
1633 | |             core::mem::size_of::<Call>() <= 20,
1634 | |         );
     | |__________^ attempt to compute `0_usize - 1_usize`, which would overflow
     |
     = note: this error originates in the macro `static_assertions::const_assert` (in Nightly builds, run with -Z macro-backtrace for more info)

```

Reply:
Until const panics are a thing, static assertions are rather useless.